### ISSUE TITLE
scalers dict 

### ISSUE THREAD

**Issue by agoodge** [TorchSpatiotemporal/tsl#45]
Hi, thank you for the very useful library.

I am wondering about this line:
``scalers = {'target': StandardScaler(axis=(0, 1))}``

1) Why do we require a dictionary of scalers, instead of passing a single scaler object?
2) Why is the key "target"? Does this mean only the target data is scaled? What about the training data?
3) What other scaler key options are available and what is the difference?

Thanks a lot!

**Comment by andreacini** [comment]
Hi,`target` refers to the `target` argument used to initialize a SpatioTemporalDataset. In other words, it refers to the variable you want to forecast (check the doc [here](https://torch-spatiotemporal.readthedocs.io/en/latest/usage/spatiotemporal_dataset.html)). 

You can use a different scaler for each covariate/attribute or, in general, for each element in a batch. For instance, in [the traffic example](https://github.com/TorchSpatiotemporal/tsl/blob/main/examples/forecasting/run_traffic_experiment.py) you could specify a scaler for the covariate `u`.

**Comment by agoodge** [comment]
"it refers to the variable you want to forecast"

Thanks for your reply. I thought this was the case. Does this mean that the target is scaled but the input is not scaled?

**Comment by andreacini** [comment]
Target means that it's the variable you want to forecast. In the standard setup, you'll have a window of past observation of the target variable as part of the input (past observations), and the next H time steps as regression targets for the output of your model. Both will be scaled,. Additionally, the `scale_target` flag in `Predictor` allows you to decide whether to compute the training loss before or after scaling back to the original scale. All metrics will be computed with respect to the original scale (as it should be).
